src/main/resources/init_db.sql

```diff
@@ -0,0 +1,17 @@
+CREATE DATABASE `book_db`;
+
+USE book_db;
```

Comment:
shouldn't you create schema (if not exists) and use it here?

I don't think that it is a good idea to create whole db in init script. in addition db is specified in the connection url

Reply:
As far as I know, in MySQL there is no difference between create schema and create database. (this is from official documentation: https://dev.mysql.com/doc/refman/8.0/en/create-database.html).
But anyway, your comment make sence I should use it with "if not exist"